static/js/components/OrderSummary.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Card } from 'react-mdl/lib/Card';
+import Button from 'react-mdl/lib/Button';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+
+import SpinnerButton from './SpinnerButton';
+import type { UIState } from '../reducers/ui';
+import { FETCH_PROCESSING } from '../actions';
+import type { Course, CourseRun } from '../flow/programTypes';
+import type { CoursePrice } from '../flow/dashboardTypes';
+import { formatPrice } from '../util/util';
+
+class SummaryPage extends React.Component {
+  props: {
+    ui:               UIState,
+    course:           Course,
+    courseRun:        CourseRun,
+    coursePrice:      CoursePrice,
+    checkout:         Function,
+    checkoutStatus?:  string,
+  };
+  getCoursePrice(): string {
+    const { coursePrice } = this.props;
+    return formatPrice(coursePrice.price);
+  }
+  getExplanationText(): string {
+    const { coursePrice } = this.props;
+    if (coursePrice.price > 0){
+      return "Clicking below with link outside of the MIT Micromasters app" +
+        "to an external website, where you will be able to complete the transaction by" +
+        "paying with a creadit card."
+    }else{
+      return "Because there is no cost to enroll in this course, when you click the button below" +
```

Comment:
There's no space at the end of this string or the beginning of the next one, so `below` and `you` will get joined together. Maybe it would be better just to return a `<p className="payment-explanation">...</p>` from this function?